integration/cli-tests/src/test/java/org/wso2/micro/integrator/cli/CliEnvironmentSetup.java

```diff
@@ -48,4 +49,9 @@ public void setupEnv() throws IOException {
             }
         }
     }
+
+    @BeforeSuite
+    public void login() throws IOException {
```

Comment:
doesn't this method needs to depends on setupEnv

Reply:
Updated.